integration-tests/README.md

```
@@ -12,23 +12,33 @@ In order to succesfully run this integration test one must:
         * `server-token`
         * `server-url`
     * `rhtap-cli-install` used by `rhtap-install` task with keys:
-        * `quay-dockerconfig-json: `
+
         * `docker_io: `
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 4; Actual: 8

(MD007, ul-indent)